### PR TITLE
Fixed test to ensure sequelize works for locations

### PR DESCRIPTION
**Pull request by jjmerino**
Replaced some tabs by spaces too.

### CODE REVIEW

**Comment by jjmerino** [comment]
@ryojiosawa @kmeurer @suprbh @jamesongamble

**Comment by jjmerino** [comment]
germTracking database must be created in travis @ryojiosawa you got this?

**Comment by ryojiosawa** [comment]
@jjmerino - the build is failing because one of the tests you added is timing out.  Can you have a look?

@kmeurer, @jamesongamble, @suprbh

**Comment by ryojiosawa** [comment]
@jjmerino - yes but my pull request is still pending.  Can you have a look? https://github.com/germTheory/germTheoryWebApp/pull/92

File: specs/server/locationModel.spec.js [hunk 1]
```diff
@@ -1,10 +1,73 @@
...
+    });
 
-})
\ No newline at end of file
+  });
+
+  beforeEach(function(){
+  });
+
+  describe('Sequelize model',function(){
```
Comment (ryojiosawa) [inline specs/server/locationModel.spec.js:28]: @jjmerino - not sure if this test is necessary.  It appears that you're testing Sequelize's create method.  Also, there's no assert in your test.  What is it that you're trying to verify here?

Comment (jjmerino) [inline specs/server/locationModel.spec.js:28]: before and beforeEach are just the fixtures.

The tests i have are specs for our schema. If we change the schema by accident they will start failing. These tests helped me make sure that Sequelize is configured as expected by our schema specs.

**Comment by jjmerino** [comment]
:(

**Pull request closed without merging**